gtsfm/densify/mvsnets/mvsUtills.py

```diff
@@ -0,0 +1,93 @@
+"""MVSNets utils class for gtsfm
+
+Authors: Ren Liu
+"""
+from typing import Dict, Any
+
+import math
+import numpy as np 
+
+class Math(object):
+    """Some mathematic methods for MVSNets"""
+    
+    @classmethod
+    def theta_ij(cls, pi: np.ndarray, pj: np.ndarray) -> float:
+        """calculate the angle between vector pi and pj
+
+        Args:
+            pi: vector in np.ndarray of [3, ] shape
+            pj: vector in np.ndarray of [3, ] shape
+
+        Returns:
+            angle in degree with float type
+        """
+        return (180.0 / math.pi) * math.acos(np.dot(pi, pj) / np.linalg.norm(pi) / np.linalg.norm(pj))
+    
+    @classmethod
+    def piecewiseGaussian(
```

Comment:
prefer `piecewise_gaussian()` rather than `piecewiseGaussian` (snake case in python instead of the camel-case from c++)